README.md

```diff
@@ -394,6 +394,11 @@ server:
   # role to assume before querying EC2 API in order to discover metadata like EC2 private DNS Name
   ec2DescribeInstancesRoleARN: arn:aws:iam::000000000000:role/DescribeInstancesRole
 
+  # AWS Account IDs to scrub from server logs. (Defaults to empty list)
```

Comment:
Do we want to give the ability to redact more than just account IDs?

Reply:
I think we could certainly add that at some point